.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
```

Comment:
Well, Cesium does have quite a few [Typed Arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays) which are an [ECMAScript 6 API for handling binary data](http://2ality.com/2015/09/typed-arrays.html) (and no immediately discernible reference to built-in typed arrays in the [ES5 spec](https://es5.github.io/))

An alternative to `"es6": true` is to specify a handful of globals in the `.eslintrc` file:

```js
"globals": {
    "Float32Array": true,
    "Float64Array": true,
    "Uint32Array": true,
```

and so on and so on...

I may be missing something obvious, but it seems that although it is true you can't use *every* element of ES6, you are using *some*.

> **Side note**: Typed arrays seems to have been added to Cesium after version 1.29 (I have not taken the time to figure *exactly* when).  I only know this because my project at work cannot use Cesium 1.33 with PhantomJS ^1.9.0 because of `Float64Array` and it's friends (even though [PhantomJS is supposed to support typed arrays](https://kangax.github.io/compat-table/es6/#phantom), BUT [does not](https://github.com/ariya/phantomjs/issues/10607))